src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
 // export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
```

Comment:
[CODE STYLE] - You are passing an object `person` to the `Person` component and then destructuring it inside the component. It would be better to destructure `person` object directly in the props. So, instead of `({ person }) => {...}` you could write `({ name, age, isMarried, partnerName, sex }) => {...}`. This way, you can avoid nesting and make your code cleaner.